sdks/go/pkg/beam/core/runtime/exec/datasource.go

```diff
@@ -125,9 +130,16 @@ func (n *DataSource) process(ctx context.Context, data func(bcr *byteCountReader
 	bcr := byteCountReader{reader: &r, count: &byteCount}
 
 	for {
+		// The SDK is currently waiting for the Runner to send data for it to be
+		// processed. Hence, the boolean is marked as true.
```

Comment:
This comment feels redundant given the line of code it's commenting, which says the same thing.

Reply:
done. thanks!